flake.nix

```
@@ -126,7 +126,7 @@
         };
       };
 
-      homeManagerModules.default = { lib, config, pkgs, ... }: {
+      homeModules.default = { lib, config, pkgs, ... }: {
```

I was too quick. We've got folks using the module as-is, let's add a little help for our friends:

```suggestion
      homeManagerModules.default = self.homeModules.default;
      homeModules.default = { lib, config, pkgs, ... }: {
```

(Note I haven't tested this, so please make sure it works ...)

---

Good call. Done.